### PR TITLE
SONAR-6678 Move computation of Duplication measures to Compute Engine

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/computation/step/DuplicationMeasuresStep.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    private Optional<Integer> getNbLinesFromLocOrNcloc(CreateMeasureContext context) {
+      Optional<Measure> lines = measureRepository.getRawMeasure(context.getComponent(), linesMetric);
+      if (lines.isPresent()) {
+        return Optional.of(lines.get().getIntValue());
+      }
+      Optional<Measure> nclocs = measureRepository.getRawMeasure(context.getComponent(), nclocMetric);
+      if (nclocs.isPresent()) {
+        Optional<Measure> commentLines = measureRepository.getRawMeasure(context.getComponent(), commentLinesMetric);
+        int nbLines = nclocs.get().getIntValue();
+        return Optional.of(commentLines.isPresent() ? nbLines + commentLines.get().getIntValue() : nbLines);
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/computation/step/DuplicationMeasuresStep.java]: MAJOR Add parentheses to make the operator precedence explicit. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS864)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by julienlancelot** [comment]
Feedback applied !

**Comment by simonbrandhof** [comment]
LGTM

**Pull request merged**